src/constants/translations/ua/filters.json

```diff
@@ -9,5 +9,5 @@
     	}
     },
 	"filtersListTitle": "Фільтри",
-	"filtersLevelsLable": "Рівні"
+	"filtersLevelsLabel": "Рівні"
```

Comment:
I suggest refactoring to `filterLevelsLabel`

Reply:
fixed!